tests/rt.sh

```diff
@@ -614,11 +614,11 @@ EOF
       fi
 
       # Set RT_SUFFIX (regression test run directories and log files) and BL_SUFFIX
-      # (regression test baseline directories) for REPRO (IPD, CCPP) or PROD (CCPP) runs
+      # (regression test baseline directories) for REPRO or PROD runs
       if [[ ${MAKE_OPT^^} =~ "REPRO=Y" ]]; then
         RT_SUFFIX="_repro"
         BL_SUFFIX="_repro"
-      elif [[ ${MAKE_OPT^^} =~ "CCPP=Y" ]]; then
+      else
         RT_SUFFIX="_prod"
         BL_SUFFIX="_ccpp"
```

Comment:
Can BL_SUFFIX be removed?

Reply:
I was thinking to do this, yes, if @junwang-noaa and you are ok. I made the change and will keep it until the last minute (before creating new baselines), however, so that I can test against the existing baselines first when it is time to commit. This way we know for sure that the PR doesn't change anything we don't expect it to change.

Reply:
As I said, this will require several changes in `fv3_conf/*` files and I would really prefer to do this separately. It's ok to remove the `_ccpp` and `_prod` suffices as suggested by Dusan, because that is a change in this file only, but renaming all tests is too much for this commit in my opinion.

Reply:
If you want to have incremental commits, that is fine, please specify that in your PR. maybe we can call this IPD removal Part1.

Reply:
Sure, I updated the issue and the two PRs.

Reply:
Also if part 2 is renaming the test. Maybe we also need to have part3 to remove the "ifdef CCPP" in fv3 code and remove the -DCCPP=ON? So far from here it is not clear to me what the "Remove IPD" means

Reply:
Please read the issue.

Reply:
The removal will be part of one of the next steps when we rename all regression tests etc.